notifications_utils/recipients.py

```diff
@@ -276,6 +276,25 @@ def rows_with_missing_data(self):
     def rows_with_message_too_long(self):
         return self._filter_rows("message_too_long")
 
+    @property
+    def rows_with_combined_variable_content_too_long(self):
+        """
+        Checks if the length of all variable, plus the length of the template content,
+        exceeds the SMS limit of 612 characters. Counts non-GSM characters as 2.
+        """
+        rows_too_long = []
+        if self.rows_as_list:
+            for row in self.rows_as_list:
+                if row.personalisation and self.template and self.template.placeholders:
```

Comment:
why do we need to check for all 3 vs row.personalisation?

Reply:
`row.personalisation` includes the variable for the `phone number` from the CSV, causing those characters to be counted toward the limit. `as_dict_with_keys` with the placeholder column names passed in ensures we're only ever counting the variables input by the user.